### PR TITLE
Translations update from Hosted Weblate

### PR DESCRIPTION
**Pull request by weblate**
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [Hedy/Level defaults](https://hosted.weblate.org/projects/hedy/level-defaults/).

It also includes following components:

* [Hedy/web-learn more](https://hosted.weblate.org/projects/hedy/web-learn-more/)

* [Hedy/Keywords](https://hosted.weblate.org/projects/hedy/keywords/)

* [Hedy/web-Start](https://hosted.weblate.org/projects/hedy/web-start/)

* [Hedy/web-Customize class](https://hosted.weblate.org/projects/hedy/web-customize-class/)

* [Hedy/Achievements](https://hosted.weblate.org/projects/hedy/achievements/)

* [Hedy/Quizzes](https://hosted.weblate.org/projects/hedy/quizzes/)

* [Hedy/Adventures](https://hosted.weblate.org/projects/hedy/adventures/)

* [Hedy/web-For teachers](https://hosted.weblate.org/projects/hedy/web-for-teachers/)

* [Hedy/web-texts](https://hosted.weblate.org/projects/hedy/web-texts/)

Current translation status:

Weblate translation status

### CODE REVIEW

File: coursedata/adventures/nb_NO.yaml [hunk 1]
```diff
@@ -63,24 +63,24 @@ adventures:
     turtle:
         levels:
             4:
-                start_code: "{print} 'Drawing figures'\nangle {is} 90\n{turn} angle\n{forward} 25\n"
-                story_text: "## Let's draw\nIn level 4 you have to use quotation marks with `{print}` and `{ask}`. Also when drawing!\n"
-                example_code: "## Example Hedy code\n\n```\n{print} 'Drawing figures'\nangle {is} 90\n{turn} angle\n{forward} 25\n{turn} angle\n{forward} 25\n```\n"
+                start_code: "{print} Tegner figurer\nvinkel {is} 90\n{turn} vinkel\n{forward} 25\n"
```
Comment (Felienne) [inline coursedata/adventures/nb_NO.yaml:66]: Hi @Hakon!

There is an error here, you removed the '' around Tegner figure, it should be: 

```
print 'Tegner figurer'
vinkel is 90
turn vinkel
forward 25
```

Probably easiest if you fix that in Weblate.

Comment (Hakon) [reply coursedata/adventures/nb_NO.yaml]: Thank you! I will fix

Comment (Hakon) [reply coursedata/adventures/nb_NO.yaml]: I think i have already gone back to fix that, could you verify for me?

**Comment by Felienne** [comment]
Hi @vovak!! Nice to "see" you here and thanks for picking up Russian!!

**Pull request merged**